translations/pencil_fr.ts

```diff
@@ -131,12 +131,12 @@
         <location filename="../app/src/actioncommands.cpp" line="655"/>
         <source>Remove selected frames</source>
         <comment>Windows title of remove selected frames pop-up.</comment>
-        <translation type="unfinished"/>
+        <translation>Enlever les trames sélectionnées</translation>
```

Comment:
Images is used for frames everywhere else, that's probably what you want to use here and immediately below.